src/components/Login.tsx

```diff
@@ -48,6 +75,7 @@ const LoginPage = () => {
                     name="username"
                     label="username"
                     value={username}
+                    autoComplete="username email"
```

Comment:
I tested in both Firefox and Chrome. Password saving works just fine without the autoComplete attribute. I don't think we need this :-)

Reply:
These auto-complete attributes on the input tags are recommended for a login form. I can imagine the browser was smart enough to infer this was a login form without them, but I suggest we leave these attributes on the inputs